dataset_loaders/data_augmentation.py

```diff
@@ -562,6 +585,43 @@ def random_transform(x, y=None,
                                     fill_mode=fill_mode,
                                     fill_constant=cvalMask,
                                     rows_idx=rows_idx, cols_idx=cols_idx))
+    flow = None
+    if return_optical_flow:
+        from skimage import io
+        flow = []
+        if compute_optical_flow:
```

Comment:
Note: add a `shared_path` parameter as well so that if the OF is missing in the *local* path you can look for it in the shared path. If it's missing in both path you compute it and save it in both paths.